content/whats-new-tinacloud/2024.11.36.mdx

```diff
@@ -0,0 +1,14 @@
+---
+versionNumber: 2024.11.36
+dateReleased: 2024-11-11T04:03:48Z
+---
+
+<!-- Release notes generated using configuration in .github/release.yml at main -->
+
+## What's Changed
+### ✨ Features
+* Add a link to how to resolve invalid site url issues by @brookjeynes-ssw in https://github.com/tinacms/tinacloud/pull/2391
+* feat: paginated branch status by @kldavis4 in https://github.com/tinacms/tinacloud/pull/2366
```

Comment:
@wicksipedia @BrookJeynes @ncn-ssw 
Does it make sense to have these **private** links on this **public** site?

Reply:
They can be removed.